website/assets/styles/pages/docs/basic-documentation.less

```diff
@@ -38,6 +38,17 @@
     }
 
   }
+  .docs-heading:hover {
+
+    .docs-link {
+      height: 16px;
+      vertical-align: middle;
+      margin-left: 8px;
+      content: url('/images/icon-link-16x16px@2x.png');
```

Comment:
typo: we don't use the `px` suffix anywhere else, could you strip that off?
![image](https://user-images.githubusercontent.com/618009/132931637-6b9b7df2-b5ae-495b-9097-ea6d1822b06a.png)

Reply:
I'm not familiar with the `content` property, but if you're sure it works as expected on our supported browsers, I'm ok with it.

It's ostensibly supported at least: https://caniuse.com/mdn-css_properties_content

I'm guessing it works with `vertical-align`, like a table cell.  TIL @rachaelshaw